driver-core/src/main/java/com/datastax/driver/core/HostConnectionPool.java

```diff
@@ -301,6 +306,7 @@ public void run() {
     List<Connection> newConnections = manager.connectionFactory().newConnections(this, toCreate);
     connections.addAll(newConnections);
 
+    boolean tracingRequested = isTracingRequested();
```

Comment:
Either at least add `final` here, or even get rid of the variable here entirely - you can call the function directly where you need this value below.